Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/InputValue.cs

```diff
@@ -45,6 +82,36 @@ public TValue Get<TValue>()
         }
 
         ////TODO: proper message if value type isn't right
+        /// <summary>
+        /// Check if the action button is pressed
+        /// </summary>
+        /// <returns>True if the button is activated over the button threshold. False otherwise</returns>
```

Comment:
Maybe check with @duckets but shouldn't it be <c>true</c> and <c>false</c>?

Reply:
I capitalised as it was the start of a sentance.